packages/forms/resources/views/components/wizard.blade.php

```diff
@@ -209,7 +209,6 @@ class="w-5 h-5"
                 x-show="! isLastStep()"
                 x-cloak
                 x-on:click="$wire.dispatchFormEvent('wizard::nextStep', '{{ $getStatePath() }}', getStepIndex(step))"
-                wire:loading.attr.delay="disabled"
```

Comment:
This line has been entirely removed?

Reply:
Yeah, `<x-filament-support::button>` defines that attribute already. It was overridden previously to remove the `.delay` modifier. Now it's the same across those two components, so it no longer needs to be set on the button in the wizard.